README.md

```diff
@@ -5,30 +5,69 @@
+| CI/CD([GitHub Actions](template/{{.input_root_dir}}/.github/) or [Azure DevOps](template/{{.input_root_dir}}/.azure/))                       | [GitHub Actions](https://docs.github.com/en/actions) or [Azure DevOps](https://azure.microsoft.com/en-us/products/devops) workflows to test and deploy ML code and assets | Ship ML code faster and with confidence: ensure all production changes are performed through automation and that only tested code is deployed to prod                                   |
 
+See the [FAQ](#FAQ) for questions on common use cases.
 
-Your organization can use the default stack as is or customize it as needed, e.g. to add/remove components or
-adapt individual components to fit your organization's best practices. See the
-[stack customization guide](stack-customization.md) for more details.
+## ML pipeline structure and devloop
 
-Using Databricks MLOps Stacks, data scientists can quickly get started iterating on ML code for new projects while ops engineers set up CI/CD and ML service state
-management, with an easy transition to production. You can also use MLOps Stacks as a building block
-in automation for creating new data science projects with production-grade CI/CD pre-configured.
+An ML solution comprises data, code, and models. These assets need to be developed, validated (staging), and deployed (production). In this repository, we use the notion of dev, staging, and prod to represent the execution
+environments of each stage. 
 
-![MLOps Stacks diagram](doc-images/mlops-stacks.png)
+An instantiated project from MLOps Stacks contains an ML pipeline with CI/CD workflows to test and deploy automated model training and batch inference jobs across your dev, staging, and prod Databricks workspaces. 
 
-See the [FAQ](#FAQ) for questions on common use cases.
+<img src="doc-images/mlops-stack-summary.png">
+
+Data scientists can iterate on ML code and file pull requests (PRs). This will trigger unit tests and integration tests in an isolated staging Databricks workspace. Model training and batch inference jobs in staging will immediately update to run the latest code when a PR is merged into main. After merging a PR into main, you can cut a new release branch as part of your regularly scheduled release process to promote ML code changes to production.
+
+### ML devloop - develop ML pipelines
+
+
+
+https://github.com/yinxi-db/mlops-stack/assets/80770057/062d1732-4854-482a-b989-2cfcdfc92f6e
```

Comment:
Can you update these references to point to the video assets that you uploaded to the repo now?

Reply:
updated!